Docs/source/Hardware/Boards.rst

```diff
@@ -68,23 +68,117 @@ A switching regulator (NCP1117) converts the +5V input to +3.3V. This does not n
 
     Leader board power distribution
 
-Fan Board: Manthan Upadhyaya
-=============================
+Fan Board
+=========
 
-Purpose
+Overview
+^^^^^^^^
+`GitHub Link <https://github.com/lhr-solar/BPS-FanPCB>`__
+
+`BOM Link <https://www.mouser.com/ProjectManager/ProjectDetail.aspx?AccessID=272bad62cd>`__ 
+
+Brief Description/Purpose:
     The purpose of this board is to set the speed of the fans that will be used to cool the battery 
-    pack. It recieves signals from the Master Board and sends them across a MOSFET that switches the 
-    fan on or off.
+    pack. It receives signals from the Leader Board and sends them across a MOSFET that switches 
```

Comment:
This is the first place we mention MOSFETs, so I think we should put a term link here